diff --git a/etc/pam.d/fingerprint-auth.pamd b/etc/pam.d/fingerprint-auth.pamd
index 604b95f..fdcbcc0 100644
--- a/etc/pam.d/fingerprint-auth.pamd
+++ b/etc/pam.d/fingerprint-auth.pamd
@@ -14,6 +14,5 @@ password    required      pam_deny.so
 
 session     optional      pam_keyinit.so revoke
 session     required      pam_limits.so
--session     optional      pam_systemd.so
 session     [success=1 default=ignore] pam_succeed_if.so service in crond quiet use_uid
 session     required      pam_unix.so
diff --git a/etc/pam.d/password-auth.pamd b/etc/pam.d/password-auth.pamd
index 2e01bf9..1f893ff 100644
--- a/etc/pam.d/password-auth.pamd
+++ b/etc/pam.d/password-auth.pamd
@@ -13,6 +13,5 @@ password    required      pam_deny.so
 
 session     optional      pam_keyinit.so revoke
 session     required      pam_limits.so
--session     optional      pam_systemd.so
 session     [success=1 default=ignore] pam_succeed_if.so service in crond quiet use_uid
 session     required      pam_unix.so
diff --git a/etc/pam.d/smartcard-auth.pamd b/etc/pam.d/smartcard-auth.pamd
index e5b57e3..07f03f3 100644
--- a/etc/pam.d/smartcard-auth.pamd
+++ b/etc/pam.d/smartcard-auth.pamd
@@ -14,6 +14,5 @@ password    optional      pam_pkcs11.so
 
 session     optional      pam_keyinit.so revoke
 session     required      pam_limits.so
--session     optional      pam_systemd.so
 session     [success=1 default=ignore] pam_succeed_if.so service in crond quiet use_uid
 session     required      pam_unix.so
diff --git a/etc/pam.d/system-auth.pamd b/etc/pam.d/system-auth.pamd
index 2e01bf9..1f893ff 100644
--- a/etc/pam.d/system-auth.pamd
+++ b/etc/pam.d/system-auth.pamd
@@ -13,6 +13,5 @@ password    required      pam_deny.so
 
 session     optional      pam_keyinit.so revoke
 session     required      pam_limits.so
--session     optional      pam_systemd.so
 session     [success=1 default=ignore] pam_succeed_if.so service in crond quiet use_uid
 session     required      pam_unix.so
